### PR TITLE
Switch to new-central

### PR DESCRIPTION
**Pull request by charleskawczynski**
This PR attempts to fix our CI.

### CODE REVIEW

**Comment by juliasloan25** [comment]
should we use new-central in .buildkite/pipeline.yml for this repo too?

**Comment by Sbozzolo** [comment]
(PR #48 is what is currently running on our cluster.)

File: bin/poll.py [hunk 1]
```diff
@@ -95,7 +95,7 @@ def all_canceled_builds():
     
     BUILDKITE_QUEUE = os.environ.get(
         'BUILDKITE_QUEUE',
-        'central'
+        'new-central'
```
Comment (charleskawczynski) [inline bin/poll.py:98]: @Sbozzolo, this change is missing in #48.

File: bin/slurmjob.sh [hunk 2]
```diff
@@ -4,7 +4,7 @@
 #SBATCH --reservation=clima
 
 BUILDKITE_PATH=${BUILDKITE_PATH:=/groups/esm/slurm-buildkite}
-BUILDKITE_QUEUE=${BUILDKITE_QUEUE:=central}
+BUILDKITE_QUEUE=${BUILDKITE_QUEUE:=new-central}
```
Comment (charleskawczynski) [inline bin/slurmjob.sh:7]: @Sbozzolo, so is this one

Comment (charleskawczynski) [reply bin/slurmjob.sh:7]: (and below in this file)

Comment (Sbozzolo) [reply bin/slurmjob.sh:7]: BUILDKITE_QUEUE is set in the cron.sh job (and has to be set carefully). The one below is just a comment. We can update them, but they don't affect anything.

**Comment by Sbozzolo** [comment]
Implemented in #48

**Pull request closed without merging**